### PR TITLE
help meeee

### PR DESCRIPTION
**Pull request by kwametsunami**

### CODE REVIEW

File: src/radioMarkup.md [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+
+
+
+
+
+
+
+
+
+                                                radioUrl === stationDetails.url_resolved
```
Comment (GregCrutchlow) [inline src/radioMarkup.md:303]: I'm guessing this code is needed?

File: src/App.css [hunk 2]
```diff
@@ -1,38 +1,177 @@
...
+
+html { box-sizing: border-box;}
+
+*, *:before, *:after {box-sizing: inherit}
+
+.sr-only { position: absolute; width: 1px; height: 1px; margin: -1px; border: 0; padding: 0; white-space: nowrap; clip-path: inset(100%); clip: rect(0 0 0 0); overflow: hidden;}
+
```
Comment (PPableo) [inline src/App.css:8]: Great implementation of page setup!

File: src/Radio.css [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+#rhap_current-time {
+    color: #ede8e7;
+}
+
+.rhap_button-clear {
+    color: #ede8e7;
+}
+
+/* .rhap_play-pause-button svg {
```
Comment (GregCrutchlow) [inline src/Radio.css]: Remove commented out code if not using.

File: src/App.css [hunk 4]
```diff
@@ -1,38 +1,177 @@
...
 .App {
   text-align: center;
+  font-family: 'Poppins', sans-serif;
```
Comment (PPableo) [inline src/App.css:11]: might be better off to use link google fonts to keep files smaller maybe

File: public/index.html [hunk 5]
```diff
@@ -2,7 +2,13 @@
...
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Michroma&display=swap" rel="stylesheet">
+
```
Comment (PPableo) [inline public/index.html:11]: I see you have a file for fonts but also linked the google fonts. Poppins should be available font for import just make sure to update the href to include Popppins

File: src/App.css [hunk 6]
```diff
@@ -1,38 +1,177 @@
...
+  border: 35px solid black;
+  height: 100%;
+  padding-bottom: 50px;
+}
+
+.wrapper {
+  max-width: 1200;
+  width: 90%;
+  margin: 0 auto;
+}
```
Comment (PPableo) [inline src/App.css:27]: great use of wrapper implementation!

File: src/Stations.js [hunk 7]
```diff
@@ -0,0 +1,112 @@
...
+                apiError
+                    ? <h2>Oh no! We're having an issue. Please refresh, then try again later.</h2>
+                    : <h2>Choose a station</h2>
+            }
+            {
+                apiError
+                ? <div>
+                    <h3>The server is currently overloaded with requests. In the meantime, here's some music to enjoy.</h3>
+                    <iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/pxw-5qfJ1dk" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+                </div>
```
Comment (PPableo) [inline src/Stations.js:57]: Like the error handling to failed requests!

**Pull request merged**